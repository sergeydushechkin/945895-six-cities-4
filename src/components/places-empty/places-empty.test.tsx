import * as React from "react";
import * as renderer from "react-test-renderer";
import PlacesEmpty from "./places-empty";

it(`Render PlacesEmpty`, () => {
  const tree = renderer
    .create(
        <PlacesEmpty />
    )
    .toJSON();

  expect(tree).toMatchSnapshot();
});
