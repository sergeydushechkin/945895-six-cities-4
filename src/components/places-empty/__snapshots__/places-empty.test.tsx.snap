// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render PlacesEmpty 1`] = `
<div
  className="cities__places-container cities__places-container--empty container"
>
  <section
    className="cities__no-places"
  >
    <div
      className="cities__status-wrapper tabs__content"
    >
      <b
        className="cities__status"
      >
        No places to stay available
      </b>
      <p
        className="cities__status-description"
      >
        We could not find any property availbale at the moment in 
        Dusseldorf
      </p>
    </div>
  </section>
  <div
    className="cities__right-section"
  />
</div>
`;
